tests/bootstrap.php

```diff
@@ -3,8 +3,29 @@
 
 use Doctrine\ORM\Tools\Setup;
 use Doctrine\ORM\EntityManager;
+use Symfony\Component\Dotenv\Dotenv;
 
-$config = parse_ini_file(__DIR__ . '/config.ini');
+$_SERVER['APP_ENV'] = 'test';
+$_SERVER['KERNEL_CLASS'] = 'App\Kernel';
+
+if (!file_exists(__DIR__ . "/../.env.test")) {
+    die('Unable to find the .env.test file');
+}
+
+$dotenv = new Dotenv();
+$dotenv->load(__DIR__ . "/../.env.test");
```

Comment:
Il faudrait tester et loader .env.test puis .env.test.local

Reply:
propositon de patch : 
<pre>
diff --git a/tests/bootstrap.php b/tests/bootstrap.php
index c7c7dd1a..c3b0976f 100644
--- a/tests/bootstrap.php
+++ b/tests/bootstrap.php
@@ -8,12 +8,21 @@ use Symfony\Component\Dotenv\Dotenv;
 $_SERVER['APP_ENV'] = 'test';
 $_SERVER['KERNEL_CLASS'] = 'App\Kernel';

-if (!file_exists(__DIR__ . "/../.env.test")) {
-    die('Unable to find the .env.test file');
+$dotenv = new Dotenv();
+
+if (!file_exists(__DIR__ . "/../.env.test.local")) {
+    die("Unable to find the .env.test file\n");
+}
+
+
+if (file_exists(__DIR__ . "/../.env.test")) {
+    $dotenv->load(__DIR__ . "/../.env.test");
+}
+
+if (file_exists(__DIR__ . "/../.env.test.local")) {
+    $dotenv->load(__DIR__ . "/../.env.test.local");
 }

-$dotenv = new Dotenv();
-$dotenv->load(__DIR__ . "/../.env.test");
 $database_url = $_ENV['DATABASE_URL'];

 $pattern = '/.[^\/]*\/\/(.[^:]*):(.[^@]*)@(.[^:]*):(\d*)\/(.*)/';
</pre>

J'ai testé avec : OK